packages/helix-shared-bounce/src/bounce.js

```diff
@@ -29,17 +32,37 @@ function bounce(func, { responder, timeout = 500 }) {
     context.invocation.bounceId = bounceId;
 
     // run the quick responder function
-    const holdingResponse = await responder(request, context);
+    const holdingResponse = (async () => {
+      const res = await responder(request, context);
+      await timer.setTimeout(timeout);
```

Comment:
how does this work?
the `responder()` call is waited, so the `timer.setTimeout()` is only executed after the responder responded.

Reply:
I think you don't need this timer at all, since you have the timeout in the fetch request.

Reply:
The responder is usually much faster than the `fetch` request, so we need to wait a bit longer to give the `fetch` request a chance to return.

The proper way to do this would be `Promise.race(timeout, Promise.all(timeout, res))`